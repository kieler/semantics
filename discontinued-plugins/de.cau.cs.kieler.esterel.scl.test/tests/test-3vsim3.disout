module test_3vsim3:
input A;
input B;
input C;
input D;
output E;
output F;
output G;

trap T in
  if (A) {
      if (B) else {
        emit E
      }
    ||
      if (C) else {
        if (D) else {
          exit T
        };
        emit G
      };
      emit E
    ||
      if (E) else {
        emit F
      }
  }
end trap

end module
