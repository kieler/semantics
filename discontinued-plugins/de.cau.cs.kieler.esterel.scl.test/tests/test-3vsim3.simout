   0 E=0 F=0 G=0 
