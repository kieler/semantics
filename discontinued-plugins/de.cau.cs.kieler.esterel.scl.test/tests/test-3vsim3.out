module test_3vsim3:
input A;
input B;
input C;
input D;
output E;
output F;
output G;

trap T in
  present A then
      present B else
        emit E
      end present
    ||
      present C else
        present D else
          exit T
        end present;
        emit G
      end present;
      emit E
    ||
      present E else
        emit F
      end present
  end present
end trap

end module
